### PR TITLE
Update repo links in docs and build.gradle

### PR DESCRIPTION
**Pull request by jonchan51**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -133,8 +133,8 @@ asciidoctor {
         idprefix: '',  // for compatibility with GitHub preview
         idseparator: '-',
         'site-root': "${sourceDir}",  // must be the same as sourceDir, do not modify
-        'site-name': 'AddressBook-Level3',
-        'site-githuburl': 'https://github.com/se-edu/addressbook-level3',
+        'site-name': 'weme',
```
Comment (gary-lgy) [inline build.gradle]: Should we capitalize?

File: build.gradle [hunk 2]
```diff
@@ -133,8 +133,8 @@ asciidoctor {
...
+        'site-githuburl': 'https://github.com/AY1920S1-CS2103-T16-3/main',
         'site-seedu': true,  // delete this line if your project is not a fork (not a SE-EDU project)
```
Comment (gary-lgy) [inline build.gradle]: Should we delete this line too? If so, the corresponding lines in `docs/templates/_header.html.slim` can also be deleted.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103-T16-3/main
```
Comment (gary-lgy) [inline docs/UserGuide.adoc]: The link in `src/main/java/seedu/weme/ui/HelpWindow.java` should also be updated.

File: build.gradle (hunk 2 above)
Comment (jonchan51) [reply build.gradle]: isnt our proj a fork

Comment (gary-lgy) [reply build.gradle]: See https://github.com/nus-cs2103-AY1920S1/forum/issues/128

**Pull request merged**